src/analytics/private/sqls/redshift/event-parameter.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS {{schema}}.{{table_event_parameter}}(
+    event_timestamp BIGINT,
+    event_id VARCHAR(255) DEFAULT RANDOM(),
+    event_param_key VARCHAR(255),
```

Comment:
should we add a field for the type of event value?

Reply:
And how do we handle with the same event parameter name in different events?

Reply:
added `event_name` to table.  the `event_id` which is a foreign key refer to event table. the `DEFAULT RANDOM()` here made the confusion, removed it.